dart/lib/src/sentry_trace_origins.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:meta/meta.dart';
+
+@internal
+class SentryTraceOrigins {
+  static const manual = 'manual';
+
+  static const autoNavigationSentryNavigatorObserver =
+      'auto.navigation.sentry_navigator_observer';
+  static const autoHttpHttpTracingClient = 'auto.http.http.tracing_client';
```

Comment:
`http` twice here?

Reply:
Oh because `http` is actually the library, makes sense, `auto.http.http.base_client` maybe?

Reply:
The instrumented class is `BaseClient`

Reply:
It's not clear to me from the documentation of the parts to what it is referring to exactly:

`integration-part: Is the part of the integration of the SDK that created the trace or span.`

I was assuming `integration of the SDK` was refering to our classes that create the spans, as SDK is referring to the sentry SDK. 🤔

Reply:
I guess it's a matter of interpretation, I get your point as well, let's see what @philipphofmann says.

Reply:
> I was assuming integration of the SDK was refering to our classes that create the spans

Well, for iOS we're using more generic origins names, kinda related to the integration, but definitely not the classes names.

But this is one of the things that dont need to be similar across SDKs, the goal is to easily know how many spans are being created per integration. As long the name make sense in the flutter project, you could choose anything.

Reply:
The integration part is optional. You don't have to send it. I opened a PR https://github.com/getsentry/develop/pull/987. Is it clearer now?

Reply:
Yeah that makes sense, i will simplify the names.